framework/src/org/checkerframework/common/basetype/messages.properties

```diff
@@ -64,6 +64,8 @@ purity.more.deterministic=the method %s could be declared as @Deterministic
 
 flowexpr.parse.index.too.big=the method does not have a parameter %s
 flowexpr.parse.error=cannot parse the expression '%s'
+# Same as above, but only issued at call sites for a method with a postcondition.
+flowexpr.parse.postcondition.error=cannot parse the expression '%s'
```

Comment:
Can the error message provide more information, to make it easier to diagnose the problem? E.g. look at postconditions of method `C.foo` to fix this.